README.md

```diff
@@ -1,20 +1,47 @@
-This repo contains a list of Engineering titles, roles, and associated responsibilities:
+The Mariana Tek Engineering Ladder defines the responsibilities and expectations of people working at various levels within the engineering team. It exists as a guide for engineers pursuing excellence and advancement, as well as a framework for managers to evaluate performance and identify candidates for promotion.
 
+Like many engineering teams, we outline two paths for career advancement: the management track and the technical track. Engineers pursuing the management track ("Managers") are interested in managing people and teams. Engineers pursuing the technical track ("Makers") are interested in honing their technical expertise and shaping the technical direction of the company.
+
+Framed another way by engineering team at [Chartbeat](https://engineering.chartbeat.com/2015/06/05/engineering-ladders/), the difference between Makers and Managers is how they answer this question:
+
+> “Do I want to build bigger and better systems, or do I want to manage bigger and better teams?”
+
+Every engineer starts their career as a maker. As you work your way from Engineer I to Engineer II to Senior Engineer, you are primarily focused on expanding your technical knowledge and mastering the skills required of a professional software engineer. When you're ready to move beyond Senior Engineer, you have the opportunity to specialize.
+
+Neither track is superior to the other, and engineers will often switch between them as they move through their careers.
```

Comment:
👍 Great! Really glad we call this fluidity out, as I think the absence of such a remark may inadvertently give an impression of limitation or restriction.

Reply:
👍 plus one on calling out the fluidity in paths